static-assets/components/cstudio-forms/forms-engine.js

```diff
@@ -2598,7 +2603,12 @@ var CStudioForms =
               }
 
               const defaultValue = moduleConfig.config.field.defaultValue;
-              if (!value && defaultValue && typeof defaultValue === 'string' && !noDefaultLookup[formField.id]) {
+              if (
```

Comment:
@rart I was wrong about the 'no-default' attribute, it is only used in type string values as we can't distinguish between empty and unset values. Though there was this issue, not only on booleans, where if you had a default value in a rep-group item and you empty the value, then add another item, it was re-setting it to the default value.

Reply:
Confirm/test this with the rest of controls that they continue to behave correctly.